app/js/demo-controller.js

```diff
@@ -92,8 +126,13 @@ export class DemoPaywallController {
               });
         }
       } else {
+        // A list of offers to show to the user, if available.
+        const offers = [];
```

Comment:
this is always supposed to be an empty array?

Reply:
i guess i'll just remove this.